install/prereqs/debian.rst

```diff
@@ -47,21 +47,21 @@ Prefix the :command:`apt-get` command with :command:`sudo` if necessary.
 
 .. note::
 
-   **New since 15.0**: The gcc compiler to be used shall support **C++ 14**. The gcc version used in Ubuntu 16 to compile the LSST Science Pipelines is **gcc 5.4**.
+   **New since 15.0**: The gcc compiler to be used shall support **C++ 14**. The lowest gcc version that supports **C++ 14** is **gcc 6.1**. The :command:`gcc` compiler installed using the above steps in an Ubuntu 18 gcc is **gcc 7.4**.
```

Comment:
Isn't this new since 17.0?

Also, "that supports **C++14** as used by LSST is **gcc 6.1**.." because GCC5 does support some of C++14.
The second sentence mentions gcc too many times.

Reply:
Simplified the phrasing. If OK I will squash the two commits.